andela/valinor-ah-frontend
# #161290964 Build local login feature

#### What does this PR do?

Have user be able to login to their Author's Haven accounts

#### Description of Task to be completed?

- add tests
- add login action creators
- add welcome page component

#### How should this be manually tested?

- run `npm start` to spin up the server
- open localhost:3000 to view the landing page
- use redux dev tool to observe the state and state history
- navigate to the login page and login with a valid user email
- go to your inbox look for the email from Author's Haven and click on the link
- you should be directed to the welcome page and logged in now

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#161290964](https://www.pivotaltracker.com/story/show/161290964)

#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-12-19 at 3 07 28 pm" src="https://user-images.githubusercontent.com/15168965/50226388-6032b800-03a3-11e9-9ea7-7d75ad4255de.png">

<img width="1440" alt="screen shot 2018-12-19 at 3 08 28 pm" src="https://user-images.githubusercontent.com/15168965/50226389-6032b800-03a3-11e9-8dd4-b0a3e0b1a9cc.png">

<img width="1440" alt="screen shot 2018-12-16 at 12 42 26 pm" src="https://user-images.githubusercontent.com/15168965/50226387-6032b800-03a3-11e9-8dd3-857215b55519.png">

<img width="1440" alt="screen shot 2018-12-19 at 3 40 13 pm" src="https://user-images.githubusercontent.com/15168965/50226813-7ab96100-03a4-11e9-9f23-0d9ba4735fa9.png">

#### Questions:

N/A

[Review App](https://valinor-ah-frontend-stag-pr-31.herokuapp.com/login)
